crates/turbo-malloc/src/counter.rs

```diff
@@ -0,0 +1,72 @@
+use std::{
+    cell::UnsafeCell,
+    ptr::NonNull,
+    sync::atomic::{AtomicUsize, Ordering},
+};
+
+pub static ALLOCATED: AtomicUsize = AtomicUsize::new(0);
+const RESERVE: usize = 1000240; // 100 KiB
+const MAX_RESERVE: usize = 2000480; // 200 KiB
```

Comment:
There's one too many zeroes.

Also, can you comment what each of these constants control?

```suggestion
const KB: usize = 1024;
const RESERVE: usize = 1024 * KB; // 100 KiB
const MAX_RESERVE: usize = 2048 * KB; // 200 KiB
```